heron/schedulers/src/java/com/twitter/heron/scheduler/utils/SchedulerConfig.java

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.twitter.heron.spi.utils;
+package com.twitter.heron.scheduler.utils;
```

Comment:
This class is really a utility, not a config. We should probably name it `SchedulerConfigUtils`. Also all but the last method can be private.